.github/workflows/validation.yaml

```diff
@@ -0,0 +1,37 @@
+name: Validations
+
+on:
+  pull_request:
+    types: [ opened, edited, synchronize ]
+  push:
+    branches:
+      - main
```

Comment:
"No cyrillic" validation is useless for main. "git diff origin/main..." will be empty.